### ISSUE TITLE
Icon Request: fa-ratings

### ISSUE THREAD

**Issue by mseminatore** [FortAwesome/Font-Awesome#13640]
An icon to represent ratings.

![image](https://user-images.githubusercontent.com/16830648/43294409-ecf5a22a-90f3-11e8-92ab-d3a778d1e9b3.png)

**Comment by Ednas** [comment]
You can accomplish that by using star https://fontawesome.com/icons/star?style=solid by combining the solid and regular star, for example if the rating was 4 stars, you'd display 4 solid stars and 1 regular star.

**Comment by mseminatore** [comment]
Yes, I thought of that and that is certainly possible.  But the sizing doesn't work very well.  For now I am using a single half-star-alt.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’re closing this issue because we think it might be too niche, we may already have similar icons, or its just not a good fit for Font Awesome.

But if you feel really strongly about it, feel free to make your case by submitting a new issue.
